bpm/bonita-api/bonita-common-api/src/main/java/org/bonitasoft/engine/form/FormMappingTarget.java

```diff
@@ -35,7 +35,12 @@
     LEGACY,
 
     /**
-     * The form mapping is not defined or not necessary
+     * The form mapping is not defined and IS necessary
```

Comment:
The form mapping is explicitly not yet defined but IS necessary for the process to be resolved